src/libraries/Microsoft.PowerFx.Core/Public/Values/FormulaValueNew.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.Public.Types;
+
+namespace Microsoft.PowerFx.Core.Public.Values
+{
+    // Basc new operators. 
+    public partial class FormulaValue
+    {
+        #region Host Utility API
+
+        // Host utility creation methods, listed here for discoverability.
+        // NOT FOR USE IN THE INTERPRETER! When creating new instances in
+        // the interpreter, call the constructor directly and pass in the
+        // IR context from the IR node.
+        public static NumberValue New(double number)
+        {
+            return new NumberValue(IRContext.NotInSource(FormulaType.Number), number);
+        }
+
+        public static FormulaValue New(double? number)
```

Comment:
Why is this one nullable, but not the others? Should we be consistent and accept nullable for all these formula values?

Reply:
Not sure - this was existing code moved from FormulaValue.cs

Reply:
Cool, added #241  to look into this